core/message-tags-3.3.md

```diff
@@ -105,7 +105,9 @@ Servers MAY apply moderation to this command using existing or newly specified m
 
 Servers MUST NOT deliver `TAGMSG` to clients that haven't negotiated the message tags capability.
 
-Servers SHOULD reject any `TAGMSG` command sent without tags. In this case, they MUST use the `ERR_NEEDMOREPARAMS` (`461`) error numeric.
+Servers SHOULD reject any `TAGMSG` command sent without tags, or where there are no tags with the client-only prefix
+and all tags without the client-only prefix are removed by the server.
```

Comment:
Ok. Thinking about this further, it should really be a MUST instead of a SHOULD.

Otherwise we just deliver useless empty tag messages and the client is unaware.